mcp/src/main/java/io/modelcontextprotocol/spec/McpSchema.java

```diff
@@ -964,61 +965,34 @@ public record CompleteCompletion(// @formatter:off
 			@JsonSubTypes.Type(value = EmbeddedResource.class, name = "resource") })
 	public sealed interface Content permits TextContent, ImageContent, EmbeddedResource {
 
-		String type();
+		// String type();
```

Comment:
It should be removed I think.

Reply:
Actually I've decided to add a `default String type()` implementation to comply with the MCP Schema, were the type in the content exist.